client/beefy/src/import.rs

```diff
@@ -116,11 +117,9 @@ where
 		}
 		// Send the justification to the BEEFY voter for processing.
 		match justification {
-			// TODO #11838: Should not unpack, these channels should also use
-			// `VersionedFinalityProof`.
-			VersionedFinalityProof::V1(signed_commitment) => self
+			BeefyVersionedFinalityProof::<Block>::V1(signed_commitment) => self
 				.justification_sender
-				.notify(|| Ok::<_, ()>(signed_commitment))
+				.notify(|| Ok::<_, ()>(BeefyVersionedFinalityProof::<Block>::V1(signed_commitment)))
```

Comment:
no need to unpack here just to pack it back in `V1`, just directly send the `justification`

Reply:
Right, I made confused here. :smile: